charts/linkerd-control-plane/values.yaml

```diff
@@ -224,6 +224,13 @@ proxyInit:
       limit: ""
       # -- Amount of ephemeral storage that the proxy-init container requests
       request: ""
+  # -- Privileged mode allows the container processes to inherit all security
+  # capabilities and bypass any security limitations enforced by the kubelet.
+  # When used with 'runAsRoot: true', the container will behave exactly as if
+  # it was running as root on the host. May escape cgroup limits and see other
+  # processes and devices on the host. 
+  # @default -- false
+  privileged: false
```

Comment:
nit: It might make sense to move this down next to the other privilege-related entries, `runAsRoot` and `runAsUser`.